Character.cpp

```diff
@@ -88,6 +85,16 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
+void restoreAndBoostStats(int& stat, int* initialLevelPtr)
+{
+    if (stat < *initialLevelPtr)
+    {
+        stat = *initialLevelPtr;
+    }
+    stat *= 1.1;
+    *initialLevelPtr = stat;
```

Comment:
You're dereferencing 3 times here.
You don't need to.
The first argument to this function is a reference.
Why can't the 2nd argument to this function also be a reference?
- [ ] revise